nifty-script

```diff
@@ -400,7 +400,9 @@ travis_for_sites() {
 
     echo "travis_fold:start:deploy"
     deploy_args=$(./prepare-deploy ${mode} ${SITE} ${TRAVIS_BUILD_NUMBER})
-    make -C ${SITE} deploy-${mode} ${deploy_args}
+    if [ $mode = "staging" ]; then
```

Comment:
I'd like to recommend a slightly different approach here. Let's leave this deployment code untouched and instead put an explicit bailout in place earlier, along the lines of this one that already exists:


    ####
    # If this is a feature branch, then we don't deploy.
    ####
    if [ "$mode" = "feature" ]; then
        echo "Travis branch (${TRAVIS_BRANCH}) not a \"master\" nor a"
        echo "\"production-\" tag so not deploying."
        return;
    fi

Where we would test for "production" and give a message stating that production deployment is currently disabled.

That will give us some explicit messaging in the log which will hopefully prevent confusion for "Why isn't anything happening when I run 'make release'?" etc.